scripts/post-promote-tasks-utils.ts

```diff
@@ -9,18 +10,19 @@ interface VersionDiff {
   version: string;
 }
 
+async function getVersions(sha: string): Promise<IndexableVersions> {
+  const url = (sha: string) =>
+    `https://raw.githubusercontent.com/ampproject/cdn-configuration/${sha}/configs/versions.json`;
+  const response = await fetch(url(sha));
```

Comment:
The `url` functions seems superfluous to me 🙃

```suggestion
  const response = await fetch(
    `https://raw.githubusercontent.com/ampproject/cdn-configuration/${sha}/configs/versions.json`
  );
```

Reply:
a refactoring side effect 😛